crates/core-subsystem/core-resolver/src/validation/arguments_validator.rs

```diff
@@ -54,6 +54,11 @@ impl<'a> ArgumentValidator<'a> {
         &self,
         field_argument_definition: &[&InputValueDefinition],
     ) -> Result<IndexMap<String, Val>, ValidationError> {
+
+        //println!("{:?}", &self.schema);
```

Comment:
@mrl5 schema is expected to be large since it contains definitions of everything exposed in GraphQL: primitive types, custom types, queries, mutations, and various input objects (predicates, order by, etc.).

See below for index.exo - schema parity.